packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -3,6 +3,9 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+# Exit on error
+set -e
```

Comment:
Have you verified that no valid code paths currently return non-zero? I remember filing a bug at one point wanting to add `set -e` to this script but needing to hold off. If we’re lucky, I documented the known cases on that issue. (No promises!)

Reply:
The only one I could get to hit was the plistbuddy error, that's why I pulled that out to check for the presence before extraction.